concordium-node/src/utils.rs

```diff
@@ -2,26 +2,20 @@
 
```

Comment:
Maybe we should move `generate_bootstrap_dns` (and its helper functions) to `concordium-node/src/bin/dnsrecordgen.rs` as that is the only place it's being used.

Reply:
I am strongly against "moving things around on the fly" as part of PRs whose purpose is different.

Or in general moving things around unless there is a clear reason for it. It just destroys everybody's mental image of where things are.

There should at some point be some analysis and revision/cleanup of the bootstrapping code, DNS resolution. Things can be moved around then.